apps/jobs/migrations/0017_add_submission_meta_attributes_field_to_jobs.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.23 on 2020-06-27 10:12
+from __future__ import unicode_literals
+
+import django.contrib.postgres.fields.jsonb
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('jobs', '0016_add_ignore_submission_boolean_field_in_submission_model'),
```

Comment:
is this also related to submission meta attributes? unclear to me how ignore_submission is related to that

Reply:
It's a dependency. It's not related.